ariadne/src/Ariadne/Wallet/Knit.hs

```diff
@@ -127,8 +127,7 @@ instance (Elem components Wallet, Elem components Core, Elem components Cardano)
             passphrase <- getPassPhraseArg
             pure (accountRef, chain, passphrase)
         , cpRepr = \(accountRef, chain, passphrase) -> CommandAction $ \WalletFace{..} -> do
-            walletNewAddress accountRef chain passphrase
-            return $ toValue ValueUnit
+            toValue . ValueAddress <$> walletNewAddress accountRef chain passphrase
```

Comment:
I'm not sure how this change is relevant to the goal of the PR, but I guess it's a good change.

Reply:
Not directly relevant indeed, but since I had to return `Address` from `walletNewAddress`, I decided that returning `ValueAddress` would be more reasonable than ignoring the result of `walletNewAddress` (this place had to be changed anyway, at least to avoid a warning).

Reply:
Oh cool, now we can generate _and_ copy address to clipboard by one button press, as I intended.